src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -170,10 +176,12 @@ ValueElement.propTypes = {
   getChildAggCmps: PropTypes.func.isRequired,
   keyField: PropTypes.string.isRequired,
   overridableId: PropTypes.string,
+  parentValue: PropTypes.string,
```

Comment:
```suggestion
  parentValue: PropTypes.string.isRequired,
```

Reply:
We are going for a different solution, where we don't specify any more the id of the element. There is no need to pass the parent anymore